R/bambu-quantify.R

```diff
@@ -23,15 +23,17 @@ bambu.quantify <- function(readClass, annotations, emParameters,
                                ncore = ncore, verbose = verbose)
     incompatibleCounts <- incompatibleCounts[data.table(GENEID = unique(mcols(annotations)$GENEID)), on = "GENEID"]
     incompatibleCounts[is.na(counts), counts := 0]
+    compatibleCounts <- calculateCPM(compatibleCounts, incompatibleCounts)
     setnames(incompatibleCounts, "counts", colnames(readClass))
```

Comment:
I'm not sure why this line is needed here? incompatableCounts is not used anymore?

Reply:
This line is to calculate CPM with incompatible counts used. The incompatible counts will still be used later